GoogleCloudPlatform/emblem
# refactor(issue-338): Miscellaneous chores for lit app

To verify locally:

Resolves #333 & #338 

* Replace string in line 26 with your api container url (`experimental/client-app/server/proxy.js`)
```
npm i
npm run build
npm run dev
```

Additional chore items include:
* Removing `axios` and using native browser `fetch` api (recommended by lit team)
* Refactoring main store connection from `campaign-list` component to `app-dashboard` container. 
* Refactoring out `action` related items into their own folder. Separate them from `reducers`
* Adding in ability to configure env variable and toggle between flask proxy api (`../website/views/routes.py`) vs node proxy (`server/proxy.js`). See `web-dev-server.config.mjs`
